Nightly GitHub Action failure: undefined reference to `spl_read'

Our nightly GitHub Action for the container image [fails](https://github.com/rpki-client/rpki-client-container/actions/runs/8027886808/job/21938543423#step:8:2191) since today with:

```
#33 88.94 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: rpki_client-main.o: in function `entity_process':
#33 88.94 /tmp/rpki-client-portable/src/main.c:676:(.text.startup+0xc56): undefined reference to `spl_read'
#33 88.94 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /tmp/rpki-client-portable/src/main.c:678:(.text.startup+0xc76): undefined reference to `spl_insert_vsps'
#33 88.94 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /tmp/rpki-client-portable/src/main.c:681:(.text.startup+0xc7e): undefined reference to `spl_free'
#33 88.95 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: rpki_client-filemode.o: in function `proc_parser_file':
#33 88.95 /tmp/rpki-client-portable/src/filemode.c:427:(.text+0x466): undefined reference to `spl_parse'
#33 88.96 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: rpki_client-output-json.o: in function `output_json':
#33 88.96 /tmp/rpki-client-portable/src/output-json.c:158:(.text+0x4cc): undefined reference to `vsp_tree_RB_MINMAX'
#33 88.96 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /tmp/rpki-client-portable/src/output-json.c:158:(.text+0x5b4): undefined reference to `vsp_tree_RB_NEXT'
#33 88.96 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: rpki_client-parser.o: in function `proc_parser_spl':
#33 88.96 /tmp/rpki-client-portable/src/parser.c:174:(.text+0xba8): undefined reference to `spl_parse'
#33 88.96 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: rpki_client-parser.o: in function `parse_entity':
#33 88.96 /tmp/rpki-client-portable/src/parser.c:871:(.text+0xc7e): undefined reference to `spl_buffer'
#33 88.96 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /tmp/rpki-client-portable/src/parser.c:872:(.text+0xc86): undefined reference to `spl_free'
#33 88.96 /usr/lib/gcc/x86_64-alpine-linux-musl/13.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: rpki_client-parser.o: in function `proc_parser_spl':
#33 88.96 /tmp/rpki-client-portable/src/parser.c:183:(.text+0x1ba2): undefined reference to `spl_free'
#33 88.99 collect2: error: ld returned 1 exit status
#33 88.99 make[1]: *** [Makefile:521: rpki-client] Error 1
```

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -69,6 +69,7 @@ rpki_client_SOURCES += rrdp_snapshot.c
 rpki_client_SOURCES += rrdp_util.c
 rpki_client_SOURCES += rsc.c
 rpki_client_SOURCES += rsync.c
+rpki_client_SOURCES += spl.c
 rpki_client_SOURCES += tak.c
 rpki_client_SOURCES += tal.c
 rpki_client_SOURCES += validate.c

--- a/update.sh
+++ b/update.sh
@@ -76,7 +76,7 @@ for i in as.c aspa.c cert.c cms.c crl.c constraints.c encoding.c extern.h filemo
 	output-json.c output-ometric.c output.c parser.c print.c \
 	repo.c rfc3779.c roa.c rpki-client.8 rrdp.c rrdp.h rrdp_delta.c \
 	rrdp_notification.c rrdp_snapshot.c rrdp_util.c rsc.c rsync.c \
-	tak.c tal.c validate.c version.h x509.c
+	spl.c tak.c tal.c validate.c version.h x509.c
 do
 	file=`basename ${i}`
 	echo Copying ${file}

